### PR TITLE
style(nrql syntax): Minor formatting fixes

### PR DESCRIPTION
**Pull request by austin-schaefer**
Some formatting consistency fixes

### CODE REVIEW

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx [hunk 1]
```diff
@@ -1812,7 +1812,7 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
 
       <CollapserGroup>
         <Collapser title={<><InlineCode>capture()</InlineCode> within a <InlineCode>SELECT</InlineCode> clause condition</>}>
-          The following will select the domain name of the website, removing https&#x3A;// and any paths following the .com
+          The following will select the domain name of the website, removing `https://` and any paths following the `.com`
```
Comment (rhetoric101) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:1815]: @austin-schaefer is the "r'" correct before "https"?

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx [hunk 2]
```diff
@@ -1588,7 +1588,7 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
     id="func-minOf"
     title={<InlineCode>minuteOf(attribute)</InlineCode>}
   >
-    Use the minuteOf() function to extract only the minute portion (that is, seconds 0 to 59) of an attribute holding a valid timestamp value.
+    Use the `minuteOf( )` function to extract only the minute portion (that is, seconds 0 to 59) of an attribute holding a valid timestamp value.
```
Comment (rhetoric101) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:1591]: @austin-schaefer are you sure you want to insert a space between these parentheses? In the Microsoft Manual of Style, I'm not seeing examples of spaces. For example, the main function is written like this: `main()`. See https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/f/function

Plus, I'm seeing other function references in the doc that don't have this space between the parentheses.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx (hunk 1 above)
Comment (austin-schaefer) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:1815]: I'm not sure, I didn't touch that part of the doc. Looking at the surrounding examples though, it looks like it's part of the regex syntax. The intro to that section says:

> Regex expressions in NRQL use Python-like syntax, `r'...'`.

**Review by rhetoric101 — APPROVED** [review]
@austin-schaefer, thanks for making these changes!

**Pull request merged**